support/cas-server-support-actions-core/src/main/java/org/apereo/cas/web/flow/logout/TerminateSessionAction.java

```diff
@@ -102,10 +107,10 @@ protected static boolean isLogoutRequestConfirmed(final RequestContext requestCo
      * @param response the response
      */
     @SuppressWarnings("java:S2441")
-    protected static void destroyApplicationSession(final HttpServletRequest request, final HttpServletResponse response) {
+    protected void destroyApplicationSession(final HttpServletRequest request, final HttpServletResponse response) {
         LOGGER.trace("Destroying application session");
         val context = new JEEContext(request, response);
```

Comment:
This is leftover code that is left here by mistake really, and I think should be removed. The ability to invalidate a session via pac4j must be injected dynamically in areas that need this done. For example, the delegation module would find this action and inject an action itself to the webflow as an exit action that destroys the session. The code here should simply invalidate an http session, perhaps optionally.

Reply:
I hesitated here, but I will revert back to what existed.